lib/amplitude_web.dart

```diff
@@ -0,0 +1,59 @@
+@JS()
+
+import 'package:js/js.dart';
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'dart:async';
+
+@JS('amplitude')
+class Amplitude {
```

Comment:
It is ok to keep this here for now. Just wonder if in future we have a long list of methods, should we move this JS interop into another file, then imported into this file?

Reply:
I think we can, will re-organize this.